5calls/app/src/main/java/org/a5calls/android/a5calls/controller/LocationActivity.java

```diff
@@ -28,12 +30,17 @@
 import org.a5calls.android.a5calls.R;
 import org.a5calls.android.a5calls.model.AccountManager;
 
+import java.io.IOException;
+import java.util.List;
+import java.util.Locale;
+
 import butterknife.BindView;
 import butterknife.ButterKnife;
 
 public class LocationActivity extends AppCompatActivity {
-
     private static final String TAG = "LocationActivity";
+
+    public static final String ALLOW_HOME_UP_KEY = "allowHomeUp";  // Allows parent activity to control the home button
```

Comment:
Please put the comment on the previous line -- both for readability and to not go over 100 char line lengths.